contrib/runners/windows_runner/windows_runner/runner.yaml

```diff
@@ -28,6 +28,31 @@
       description: Username used to log-in.
       required: true
       type: string
+  output_schema:
+    stdout:
```

Comment:
Same here, shouldn't it always be string?

Reply:
Same as above. https://github.com/StackStorm/st2/pull/4305#discussion_r216313660

Reply:
If I'm reading this correctly it's all output for every runner.
https://github.com/StackStorm/st2/blob/fd9c6f59887efe588ecbce562ac0d80b491edd6a/st2actions/st2actions/container/base.py#L129